CHANGES.txt

```diff
@@ -5,6 +5,10 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Added the ``CREATE REPOSITORY``, ``DROP REPOSITORY``, ``CREATE SNAPSHOT``,
+   ``RESTORE SNAPSHOT`` statements and the ``sys.snapshots`` and
+   ``sys.repositories`` virtual tables for backup and restore functionality.
+
```

Comment:
i think its enough to say that the snapshot functionality for backup and restore is implemented.  since the documentation has more information